modules/ROOT/pages/aura-graphql/api-creation.adoc

```diff
@@ -0,0 +1,161 @@
+If you need multiple entries, select **Add allowed origin**.
++
+image::aura-graphql/create/cors-policy.png[]
++
+. Authentication providers
++
+All requests to the GraphQL API are authenticated and there are two options for the type of authentication: API key or JSON Web Key Set (JWKS).
+It is possible to use these in combination and have multiples of each.   
++
+After choosing the authentication provider from the dropdown list, enter a friendly name.
+This is all that is needed when using API key.
+The API key will be shown after the GraphQL API key has been created.
+If you are using JWKS, provide a friendly name and the URL from your identity provider that is used to verify JWKS tokens. 
++
+Add more more providers via **Add authentication provider**.
++
+An authentication provider can be removed by selecting the trash icon.
++
+image::aura-graphql/create/auth-providers.png[]
++
+[NOTE]
+====
+It is not recommended to use API keys with browser-based applications that are using the GraphQL API.
+This is due to the API key being visible when using browser developer tooling and very little control over GraphQL operations.
+A 3rd party identity provider with JWKS provides better security and allows for granular security rules, based on information within the JWKS token, to be defined within the Type Definitions to control GraphQL operations.
+====
++
+. Sizing
++
+Sizing a GraphQL API is based on the complexity of the type definitios, roughly their size, and the workload that is serviced.
+Larger sizes have a higher hourly cost.
+If the cost is not displayed, refer to the agreement you have with Neo4j. 
++
+image::aura-graphql/create/sizing.png[]
++
+. Creation
++
+At this point the configuration is complete for the GraphQL API.
+Select **Create** to proceed and create the GraphQL API.
+Alternatively, **Cancel** the process at top right of the page.
++
+. API key
++
+If you chose to use an API key with the GraphQL API it is now displayed.
+Store this information securely as it cannot be retrieved again.
+Select **I understand** and then **Close** once you have done this.
++
+image::aura-graphql/create/save-api-key.png[]
++ 
+. The GraphQL API will now be provisioned.  
+You can see the status via **Data APIs** from the left side navigation.
+When the status changes to "Ready", the API is ready to use.
+If it has a status of 'Error', use the three dots icon and **Edit** from the menu to change the configuration.
+
+
+== Aura CLI
+
+Like the Console, the Aura CLI can also be used to create a GraphQL API.
+There is a difference: only an API Key is used for the authentication provider.
+This can be changed in the Console after the GraphQL API has been provisioned.
+
+To create a GraphQL API with the Aura CLI, proceed as follows.
+
+. Find the AuraDB instance ID 
++
+From the table, locate the ID of the Aura instance that the GraphQL API will be used with.
++
+[source, bash, indent=0]
+----
+aura-cli instance list --output table
```

Comment:
For simplicity's sake should we omit the output flag here. If someone wants to change the output format they can use the cli docs

```suggestion
aura-cli instance list
```